airbyte-integrations/connectors/source-hubspot/source_hubspot/manifest.yaml

```diff
@@ -1658,6 +1658,170 @@ streams:
   - "#/definitions/ticket_pipelines_stream"
   - "#/definitions/workflows_stream"
 
+dynamic_streams:
+  - type: DynamicDeclarativeStream
+    stream_template:
+      type: StateDelegatingStream
+      name: custom_object_stream_name
+      $parameters:
+        entity: "custom_entity"
+        schema_properties: "schema_properties_placeholder"
+        schema_loader:
+          type: CustomSchemaLoader
+          class_name: source_hubspot.components.HubspotCustomObjectsSchemaLoader
+      full_refresh_stream:
+        name: custom_object_stream_name
+        primary_key:
+          - id
+        $ref: "#/definitions/stream_base"
+        retriever:
+          $ref: "#/definitions/base_retriever"
+          requester:
+            $ref: "#/definitions/base_requester"
+            path: "/crm/v3/objects/{{ parameters['entity'] }}"
+            request_parameters:
+              archived: "false"
+              properties:
+                type: QueryProperties
+                property_list:
+                  - "placeholder_for_properties"
```

Comment:
This should be replaced by the `QueryPropertiesFromEndpoint` I assume?

I think the `custom_properties` are defined in the config, but is the challenge here about how we resolve which properties for the current stream we are on since a user might have multiple custom objects?